.github/workflows/book.yml

```diff
@@ -16,10 +16,7 @@ jobs:
           override: true
```

Comment:
I considered switching to `dtolnay/rust-toolchain`, but it [will not support `rust-toolchain`](https://github.com/dtolnay/rust-toolchain/issues/12), and [because it uses `default` instead of `override` it doesn't work with repos that have a `rust-toolchain.toml`](https://github.com/dtolnay/rust-toolchain/issues/29).

Reply:
A few hours after I posted this, some changes were made to `dtolnay/rust-toolchain` that now make this workflow possible without manual hackery. I'll open another PR to implement it.